client/components/card/index.jsx

```diff
@@ -40,7 +40,7 @@ export default React.createClass( {
 
 		return React.createElement(
 			element,
-			assign( {}, this.props, { className } ),
+			assign( {}, omit( this.props, 'compact' ), { className } ),
```

Comment:
In fact, to clean this up a bit, you could rewrite it as:

```js
{ ...omit( this.props, 'compact', 'tagName' ), className },
```

Note also I've included `tagName`. Don't we want to `omit` that too, since it's not a valid DOM prop?